### DISCUSSION TITLE
Role of --num_cycle and --num_predictions_per_model

### DISCUSSION THREAD

**Discussion by loggy01** [KosinskiLab/AlphaPulldown#249] [Q&A] [answered]
Hi all,

Are --num_cycle and --num_predictions_per_model equivalent to --num_recycle and --num_multimer_predictions_per_model from standard AlphaFold-multimer?

Thanks in advance.

**Comment by dingquanyu** [comment, marked as answer]
Hi @loggy01 

Yep you're right. Sorry our naming is a bit confusing but they are the same as those flags in AlphaFold-Multimer.

Cheers
